extensions/podman/src/registry-setup.ts

```diff
@@ -49,7 +49,7 @@ export class RegistrySetup {
     return path.resolve(podmanConfigContainersPath, 'auth.json');
   }
 
-  protected async updateRegistries(extensionContext: extensionApi.ExtensionContext): Promise<void> {
+  protected async updateRegistries(_extensionContext: extensionApi.ExtensionContext): Promise<void> {
```

Comment:
What is the rationale for this ?

Reply:
either
1. I remove extensionContext attribute all over the calls
1. or as it's currently not used I make it optional (we could need it when using the context)

I choose option 1

Reply:
@jeffmaury you prefer option 1 ?

Reply:
@jeffmaury updated by removing unused extensionContext

Reply:
@jeffmaury I've proceed with option 1